.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    steps:
+    - name: Checkout Branch
+      uses: actions/checkout@v1
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16.20.1'
+    - name: Build Extension
+      run: |
+        npm install
+        npm run compile
+    - name: Smoke test (Linux)
+      run: xvfb-run -a npm run test
+      if: runner.os == 'Linux'
+    - name: Smoke test (Mac, Windows)
+      run: npm run test
+      if: runner.os != 'Linux'
+    # - name: Upload coverage reports to Codecov
```

Comment:
🧋 If this is not needed worth to remove it from this drop, unless there is a specific reasons for this commented out thing.

Reply:
I will remove it and add later in another PR if required really.